content/constitution.html

```diff
@@ -0,0 +1,20 @@
+---
+title: "Constitution"
+date: 2020-06-03T01:34:13+10:00
+draft: false
+mdconvert: true
+---
+
+<div class="section columns is-centered">
+    <section class="column is-6">
+        <div class="content">
+            <h1>
+                This is UQCS' constitution; any changes requested may be voted on at an Annual General Meeting. The
```

Comment:
Explain what it is i.e. the set of rules that govern the running of the society.